cmd/steal.go

```diff
@@ -51,11 +51,10 @@ func RunSteal(opts *StealOptions) (err error) {
 
 	log.Info("Stealing...")
 
-	failOnError(
-		target.Dump(),
-		"Error while dumping",
-	)
+	done := make(chan bool)
```

Comment:
you can use empty struct{} for channel, it allocates 0 K on the heap

Reply:
You can look here you want https://github.com/rafaeljesus/rabbus/blob/master/rabbus.go#L305